.buildkite/pipelines/periodic.yml

```diff
@@ -0,0 +1,52 @@
+steps:
+  - group: java-fips-matrix
+    steps:
+      - label: "{{matrix.ES_RUNTIME_JAVA}} / {{matrix.GRADLE_TASK}} / java-fips-matrix"
+        command: .ci/scripts/run-gradle.sh -Dbwc.checkout.align=true -Dtests.fips.enabled=true $$GRADLE_TASK
+        timeout_in_minutes: 180
+        matrix:
+          setup:
+            ES_RUNTIME_JAVA:
+              - openjdk17
+            GRADLE_TASK:
+              - checkPart1
+              - checkPart2
+              - checkPart3
```

Comment:
Note: However we handle branches, we need to ensure that this task is excluded in 7.17 since it doesn't exist there.

Reply:
Yep, this file just won't include the task on the 7.17 branch. The slightly more complicated issue would be if the 7.17 branch wasn't supposed to have the `periodic.yml` pipeline at all.